webtasks/cron_backend_mongodb.js

```diff
@@ -319,6 +330,71 @@ router.put('/:container/:name',
             .then(res.json.bind(res), next);
 });
 
+router.put('/:container/:name/state',
+    ensure('headers', ['host']),
+    ensure('params', ['container', 'name']),
+    function (req, res, next) {
+        var validStates = ['active', 'inactive'];
+        var data = req.webtaskContext.data;
+        var jobs = req.mongo.collection(data.JOB_COLLECTION);
+        var cluster_host = data.CLUSTER_HOST;
+        var state = req.body.state || req.query.state;
```

Comment:
It is not a required parameter. A reasonable default of `active` has been set in 86f6c14 in the `validateStates` middleware.